service_container/tags.rst

```diff
@@ -245,18 +251,18 @@ Register the Pass with the Container
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In order to run the compiler pass when the container is compiled, you have to
-add the compiler pass to the container in the ``build()`` method of your
-bundle::
+add the compiler pass to the container in a :doc:`bundle extension </bundles/extension>`
+or from your kernel::
 
-    // src/AppBundle.php
-
-    // ...
-    use Symfony\Component\DependencyInjection\ContainerBuilder;
+    // src/Kernel.php
     use App\DependencyInjection\Compiler\MailTransportPass;
+    // ...
 
-    class AppBundle extends Bundle
+    class Kernel extends Kernel
```

Comment:
`namespace App;` + `use ... as BaseKernel` + `BaseKernel`?

Reply:
fixed in 1a3be4386f60f9c2e88995406bf82b12397d8759